spec/shoryuken/middleware/chain_spec.rb

```diff
@@ -58,7 +58,9 @@ def self.success?
   it 'patches deprecated middleware' do
     subject.clear
 
-    expect(Shoryuken.logger).to receive(:warn).with("[DEPRECATION] DeprecatedMiddleware#call(worker_instance, queue, sqs_msg) is deprecated. Please use DeprecatedMiddleware#call(worker_instance, queue, sqs_msg, body)")
+    expect(Shoryuken.logger).to receive(:warn) do |&block|
+      expect(block.call).to eq("[DEPRECATION] DeprecatedMiddleware#call(worker_instance, queue, sqs_msg) is deprecated. Please use DeprecatedMiddleware#call(worker_instance, queue, sqs_msg, body)")
```

Comment:
Line is too long. [197/120]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.